src/hooks/useState.ts

```diff
@@ -0,0 +1,31 @@
+import { updateSchedule } from "../dom/render";
+import { workInProgressHookFn } from "./hooks";
+
+const useState = <T>(
+  initialState: T | (() => T)
```

Comment:
[P3] 값이 안들어오는 경우도 존재할거라 옵셔널로 두는 것이 react와 더 비슷할 것 같습니다!
```suggestion
  initialState?: T | (() => T)
```